DOCS-6157-added-links (#8)
mulesoft/docs-service-mesh: modules/ROOT/pages/manage-apis-with-service-mesh.adoc

> endif::[]

Reviewer: HIDDEN: See retitling suggestion below

= Managing APIs with Anypoint Service Mesh
ifndef::env-site,env-github[]
include::_attributes.adoc[]

Reviewer: It looks to me like this topic is really not so much about managing as securing....

:keywords: service mesh, microservices, manage APIs

image::manage-service-mesh-breadcrumb.png[]

> After you configure Service Mesh, you are now ready to manage your Mule and non-Mule applications from API Manager. You can apply policies to ensure various levels of security for your microservices. 

Reviewer: ```suggestion
After you configure Service Mesh, you can manage both Mule and non-Mule applications from Anypoint API Manager, including applying policies to secure your microservices. 
```

> For example, you can apply the Rate Limiting policy to define the number of requests that you want your service to receive within a specified period of time.

Reviewer: Nice!

To apply policies:

> . Go to *API Manager* and select the specific API instance.

Reviewer: This is so minor, but consider using "Navigate to" instead of "Go to"?

Reviewer: EDIT OF HIDDEN TEXT
Omit followup sentence to first step. Then...:

. From the left navigation, select *Policies* > *Apply New Policy*:
+
image::select-policies-for-service-mesh.png[]
+

. Select any of the following policies to secure your APIs:

* Client ID Enforcement
* JWT Validation
* Rate Limit
* Rate Limit SLA-Based

image::apply-client-enforcement-policy-service-mesh.png[]

For more information about how to apply policies, see the documentation for the specific policy.

== See Also

xref:api-manager::client-id-based-policies.adoc[Client ID Enforcement policy]
xref:api-manager::policy-mule4-jwt-validation.adoc[JWT Validation policy]
xref:api-manager::rate-limit-1.2.0-task.adoc[Rate Limiting policy]
